extensions/amp-story/1.0/amp-story.js

```diff
@@ -2507,6 +2507,7 @@ export class AmpStory extends AMP.BaseElement {
    */
   onMutedStateUpdate_(isMuted) {
     isMuted ? this.mute_() : this.unmute_();
+    this.element.toggleAttribute('muted', isMuted);
```

Comment:
Nit: please add the `muted` to the list of attributes:

https://github.com/ampproject/amphtml/blob/fb2673955f29d1279aba2ff0233c4629a7b6752a/extensions/amp-story/1.0/amp-story.js#L150-L164